packages/ifpack2/doc/UsersGuide/options.tex

```diff
@@ -376,6 +376,22 @@ \section{Chebyshev}\label{s:Chebyshev}
     % {bool}
     % {\false}
     % {If true, use the textbook variant; otherwise, use the \ifpack{} variant.}
+\ccc{chebyshev: kind}
```

Comment:
Why "chebyshev: kind" instead of "chebyshev: algorithm"?

Reply:
Hm. Yeah, I see. Before you make changes, let's see if @jhux2 or @csiefer2 have strong preferences. I don't think "kind" is very descriptive, but neither is "algorithm".

Reply:
Changed to "algorithm" -- this is a pretty easy find/replace if we want to change it before merging this PR.